packages/eslint-plugin-react-hooks/README.md

```diff
@@ -20,20 +20,37 @@ yarn add eslint-plugin-react-hooks --dev
 
 ### Flat Config (eslint.config.js|ts)
 
-For [ESLint 9.0.0 and above](https://eslint.org/blog/2024/04/eslint-v9.0.0-released/), add the `recommended` config.
+#### >= 6.0.0
+
+For users of 6.0 and beyond, simply add the `recommended` config.
+
+```js
+import * as reactHooks from 'eslint-plugin-react-hooks';
+
+export default [
+  // ...
+  reactHooks.configs.recommended,
+];
+```
```

Comment:
Hmm, I think it might be confusing to have instructions for 6.0.0 when it's not released yet. I'd suggest removing this altogether and we can add it back just as we're about to publish.

Reply:
Updated.  Do you want me to go ahead and draft the follow-up change after this merges?

Reply:
Sure!